recipes/3p_integrations/llamaindex/dlai_agentic_rag/README.md

```diff
@@ -0,0 +1,11 @@
+# Building Agentic RAG with Llamaindex
+
+The folder here containts the Llama 3 ported notebooks of the DLAI short course [Building Agentic RAG with Llamaindex](https://www.deeplearning.ai/short-courses/building-agentic-rag-with-llamaindex/).
+
+1. [Building Agentic RAG with Llamaindex L1 Router Engine](../../../quickstart/agents/dlai/Building_Agentic_RAG_with_Llamaindex_L1_Router_Engine.ipynb)
```

Comment:
The folder has only 3 notebooks, did we miss one? :)

Reply:
No we didnt :) - the first one with link in the README is in the quickstart folder: [Building Agentic RAG with Llamaindex L1 Router Engine](https://github.com/meta-llama/llama-recipes/blob/dlai_llamaindex_agent/recipes/quickstart/agents/dlai/Building_Agentic_RAG_with_Llamaindex_L1_Router_Engine.ipynb)